game.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require_relative 'environment' # #mando a llamar el archivo
+
+class Game
+  attr_accessor :depot, :control, :human
+
+  include Environment
+
+  def initialize
+    @depot = Environment::Depot.new
+    @control = Environment::Control.new
+    @board = { players: {}, control: {}, status: nil }
+  end
+
+  def new_worker(name:)
+    worker = Environment::Worker.new(name: name, personal_data: nil, professional_data: nil)
```

Comment:
Nice, I would avoid the parenthesis usage, but it is very nice you use only one line to do this.